tidy3d/web/webapi.py

```diff
@@ -322,7 +273,10 @@ def download(task_id: TaskId, path: str = "simulation_data.hdf5") -> None:
         Download path to .hdf5 data file (including filename).
 
     """
-    _download_file(task_id, fname="output/monitor_data.hdf5", path=path)
+    task = SimulationTask.get(task_id)
```

Comment:
how about SimulationTask(task_id) to set the id value.
and leave get no change?

Reply:
@lei-flex  I think in some functions (like download, start) it is ok to just set the id value as the `SimulationTask` methods don't depend on the `Simulation` or other information.

However, for `get_info` in webapi monitor, we need to pass `task.dict()`, which will be empty if we just initialize the `SimulationTask` with the id only. 

I'm not sure how to get around this. Maybe we need to refactor `get_info`? What do you think?

Reply:
yes. we could refactor it to avoid such extra webapi call.
but for first version, I think we should be fine.:)